backend/src/main/java/com/festago/dto/FestivalDetailResponse.java

```diff
@@ -0,0 +1,28 @@
+package com.festago.dto;
+
+import com.festago.domain.Festival;
+import com.festago.domain.Stage;
+import java.time.LocalDate;
+import java.util.List;
+
+public record FestivalDetailResponse(Long id,
+                                     String name,
+                                     LocalDate startDate,
+                                     LocalDate endDate,
+                                     String thumbnail,
+                                     List<FestivalDetailStageResponse> stages) {
```

Comment:
저희 이 부분 컨벤션을 논의해야 할 것 같네요!
```suggestion
public record FestivalDetailResponse(
    Long id,
    String name,
    LocalDate startDate,
    LocalDate endDate,
    String thumbnail,
    List<FestivalDetailStageResponse> stages) {
```
저는 이런 표현이 더 가독성이 높다고 생각이 들어서요!

Reply:
Id 가 한 줄로 붙어 있으니까 1번의 첫 번째 인자가 잘 안보이는 부분이 있긴하네요

Reply:
- #148 
조만간 회의를 잡아서 위 이슈에서 처리하면 좋을 것 같아요.
현재 정의되어있는 응답 객체들도 제각기라서, 한 번에 통일하는게 좋아보여요!